core/sql/executor/ExExeUtilVolTab.cpp

```diff
@@ -554,7 +554,12 @@ short ExExeUtilCleanupVolatileTablesTcb::work()
 		retcode = 0;
 
                 if ((strlen(errorSchemas_) + strlen(schemaName)) < 1000)
-                  strcat(errorSchemas_, schemaName);
+                  {
+                    strcat(errorSchemas_, schemaName);
+                    strcat(errorSchemas_, " ");
+                  }
+                else
+                  strcat(errorSchemas_, "..."); // could not fit
```

Comment:
We would want to do this at most once. As written we could keep appending dots and have a buffer overflow. (Apologies; I know this change was in response to one of my earlier comments.)